src/resources/js/v2/tickets-commerce.js

```diff
@@ -131,12 +169,15 @@ tribe.tickets.commerce = {};
 	 * @return {void}
 	 */
 	obj.bindCheckoutEvents = function( $container ) {
-		$document.trigger( 'beforeSetup.tribeTicketsCommerceCheckout', [ $container ] );
+		$document.trigger( 'beforeSetup.tecTicketsCommerce', [ $container ] );
 
 		// Bind container based events.
 		obj.bindCheckoutItemDescriptionToggle( $container );
 
-		$document.trigger( 'afterSetup.tribeTicketsCommerceCheckout', [ $container ] );
+		// Bind loader visibility.
+		obj.bindLoaderEvents();
+
+		$document.trigger( 'afterSetup.tecTicketsCommerce', [ $container ] );
```

Comment:
Maybe as a suggestion, to add more clarity, and to make it explicit that the trigger corresponds to the generic before and after setup, I'd create a method called `bindEvents` with the two triggers, and calling `bindLoaderEvents` and `bindCheckoutEvents` and just keeping `bindCheckoutItemDescriptionToggle` in `bindCheckoutEvents`.

Reply:
@juanfra Sorry, I am not clear about this one. `bindLoaderEvents` is part of the checkout and that's why it's within the `bindCheckoutEvents `. Which generic triggers you are referring to?